app/models/metric/helper.rb

```diff
@@ -121,7 +121,7 @@ def self.sanitize_start_end_time(interval, interval_name, start_time, end_time)
 
   def self.remove_duplicate_timestamps(recs)
     if recs.respond_to?(:klass) # active record relation
-      return recs unless recs.klass.kind_of?(Metric) || recs.klass.kind_of?(MetricRollup)
+      return recs unless recs.klass <= Metric || recs.klass <= MetricRollup
     elsif recs.empty? || !recs.all? { |r| r.kind_of?(Metric) || r.kind_of?(MetricRollup) }
       return recs 
```

Comment:
@jameswnl I just noticed this looking at the diff in git, can you remove the tailing whitespace here?  It was added in 9dab1c43579